### ISSUE TITLE
Date/timestamp type parsing

### ISSUE THREAD

**Issue by shalindperer** [DaredevilOSS/sqlc-gen-csharp#94]
Hi,

I’m having an issue with the timestamp type. It seems like it is not being parsed properly. Below, I have the 'created' column set as a timestamp, but the output file types it as a string. The same issue occurs with the date type.

Any help would be greatly appreciated.

I’m using v0.12.2 and PostgreSQL.

```create table if not exists appUser
(
    user_id        serial primary key,
    email          varchar(255) not null unique,
    passwordHashed varchar(255) not null,
    created        timestamp 
);


public readonly record struct GetUserArgs(string Email);
    public async Task<GetUserRow?> GetUser(GetUserArgs args)
    {
        {
            await using var connection = NpgsqlDataSource.Create(connectionString);
            await using var command = connection.CreateCommand(GetUserSql);
            command.Parameters.AddWithValue("@email", args.Email);
            var reader = await command.ExecuteReaderAsync();
            if (await reader.ReadAsync())
            {
                return new GetUserRow
                {
                    User_id = reader.GetInt64(0),
                    Email = reader.GetString(1),
                    Passwordhashed = reader.GetString(2),
                    Created = reader.IsDBNull(3) ? null : reader.GetString(3)
                };
            }

            return null;
        }
    }

**Comment by SockworkOrange** [comment]
Working on a fix

**Comment by SockworkOrange** [comment]
Fixed in release v0.12.3 :)
